docs/core/tutorials/netcore-hosting.md

```diff
@@ -19,40 +19,127 @@ Because hosts are native applications, this tutorial will cover constructing a C
+* `coreclr_create_delegate`: Creates a function pointer to a managed method
+* `coreclr_shutdown`: Unloads AppDomains and shuts down the .NET Core runtime
+* `coreclr_shutdown_2`: Like `coreclr_shutdown`, but also retrieves the managed code's exit code
+
+After loading the CoreCLR library, the next step is to get references to these functions using `GetProcAddress` (on Windows) or `dlsym` (on Linux/Mac).
+
+[!code-cpp[CoreClrHost#2](../../../samples/core/hosting/HostWithCoreClrHost/SampleHost.cpp#2)]
+
+### Step 3 - Prepare AppDomain properties
+
+Before starting the runtime, we need to prepare properties that will specify AppDomain behavior (especially concerning the assembly loader). The properties are an array of keys and a corresponding array of values that are used when starting the runtime and default AppDomain.
+
+Common AppDomain properties include:
+
+* `TRUSTED_PLATFORM_ASSEMBLIES` This is a list of assembly paths (delimited by ';' on Windows and ':' on Linux) which the AppDomain should prioritize loading and give full trust to (even in partially-trusted domains). This list is meant to contain 'Framework' assemblies and other platform assemblies, similar to the global assembly cache (GAC) in .NET Framework scenarios. Some hosts will put any library next to *coreclr.dll* on this list, others have hard-coded manifests listing trusted assemblies for their purposes.
+* `APP_PATHS` This is a list of paths to probe in for an assembly if it can't be found in the trusted platform assemblies (TPA) list. These paths are meant to be the locations where users' assemblies can be found. Common values for this property include the path the target app was loaded from or other locations where user assets are known to live.
+*  `APP_NI_PATHS` This list is very similar to APP_PATHS except that it's meant to be paths that will be probed for native images.
+*  `NATIVE_DLL_SEARCH_DIRECTORIES` This property is a list of paths the loader should probe when looking for native DLLs called via p/invoke.
+*  `PLATFORM_RESOURCE_ROOTS` This list includes paths to probe in for resource satellite assemblies (in culture-specific sub-directories).
+
+In our sample host, the TPA list is constructed by simply listing all libraries in the current directory:
+
+[!code-cpp[CoreClrHost#7](../../../samples/core/hosting/HostWithCoreClrHost/SampleHost.cpp#7)]
+
+Because the sample is simple, we only need the `TRUSTED_PLATFORM_ASSEMBLIES` and `APP_PATHS` properties:
+
+[!code-cpp[CoreClrHost#3](../../../samples/core/hosting/HostWithCoreClrHost/SampleHost.cpp#3)]
+
+### Step 4 - Start the runtime and default AppDomain
+
+Now we're ready to initialize the .NET Core runtime. Unlike the mscoree.h hosting API (described below), CoreCLRHost.h APIs start the runtime and create the default AppDomain all with a single call. We use `coreclr_initialize`, specifying a base path, name, and properties for the default AppDomain and receiving back the host handle and AppDomain ID.
```

Comment:
Remove usage of the 'we' pronoun.

Reply:
And remove the AppDomain mention.

Reply:
Removed 'we' and limited AppDomain to a single mention that the default AppDomain is automatically created as part of coreclr_initialize (instead of via a separate call as with `ICLRRuntimeHost2`).